contrib/http_servlet/src/test/java/io/opencensus/contrib/http/servlet/OcHttpServletFilterTest.java

```diff
@@ -116,17 +114,16 @@ public void testInitWithOptions() throws IOException, ServletException {
     when(mockConfig.getServletContext()).thenReturn(mockServletContext);
     when(mockServletContext.getAttribute(OC_TRACE_PROPAGATOR)).thenReturn(b3Propagator);
     when(mockServletContext.getAttribute(OC_EXTRACTOR)).thenReturn(customExtractor);
-    when(mockServletContext.getAttribute(OC_PUBLIC_ENDPOINT)).thenReturn(publicEndpoint);
+    when(mockServletContext.getInitParameter(OC_PUBLIC_ENDPOINT)).thenReturn("false");
 
     filter.init(mockConfig);
 
     verify(mockConfig).getServletContext();
-    verify(mockServletContext, times(3)).getAttribute(stringArgumentCaptor.capture());
+    verify(mockServletContext, times(2)).getAttribute(stringArgumentCaptor.capture());
```

Comment:
Add verfity(mockServletContext).getInitParameters() here as well.

Reply:
Changed as suggested in latest push